kratos/utilities/element_size_calculator.cpp

```diff
@@ -116,6 +116,105 @@ double ElementSizeCalculator<2,3>::MinimumElementSizeDerivative(
     KRATOS_CATCH("");
 }
 
+// Triangle2D6 version.
+template<>
+double ElementSizeCalculator<2,6>::MinimumElementSize(const Geometry<Node<3> >& rGeometry)
+{
+
+    /* Calculate node-edge distances */
+    const double x10 = rGeometry[1].X() - rGeometry[0].X();
+    const double y10 = rGeometry[1].Y() - rGeometry[0].Y();
+    const double x20 = rGeometry[2].X() - rGeometry[0].X();
+    const double y20 = rGeometry[2].Y() - rGeometry[0].Y();
+
+    // node 0, edge 12
+    double nx = -(y20-y10);
```

Comment:
const when possible

Reply:
These variables cannot be `const` since are modified below